src/utils/dates.js

```diff
@@ -10,3 +10,22 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+function convertTimestamp(timestamp) {
+	if (timestamp) {
+		const { seconds, nanoseconds } = timestamp;
+
+		const timestampInMilliseconds = seconds * 1000 + nanoseconds / 1e9;
+
+		return timestampInMilliseconds;
+	}
+}
```

Comment:
Great job figuring out the conversion.

I do think this could be simplified a bit by leaning on the firebase npm package. See comment below.